README.md

```diff
@@ -210,6 +210,7 @@ https://www.notion.so/Homebase-Alpha-Docs-0f0e22f3adcd4e9d87a13440ab0c7a0b
 
 ```bash
 yarn install
+yarn build dev
```

Comment:
why is this needed?

also, did you mean `yarn build:dev` or `yarn build`?

Reply:
I've been using `yarn build dev`, but `yarn build` might be sufficient.

I need this because otherwise dist doesn't get built and I get a bunch of errors when I open localhost:3000

Reply:
I'll dig into it more, this might not have been what fixed the issue.

Reply:
You shouldn't need dist to run the dev server... And yarn test already runs yarn build

Reply:
gotcha, I'll close this then and next time this happens I'll try to figure out what is breaking the build. Might be a caching thing. Thanks!